### PR TITLE
LPS-190170

### PR DESCRIPTION
**Pull request by CaiocSiqueira**
https://liferay.atlassian.net/browse/LPS-190170

PASSED Testing on: https://github.com/CaiocSiqueira/liferay-portal/pull/62

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/elasticsearch/Elasticsearch7.testcase [hunk 1]
```diff
@@ -20,6 +20,21 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-177666"
+	@priority = 3
+	test AssertDefaultConcurrentReindex {
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/elasticsearch/Elasticsearch7.testcase:25]: Rename to `CanSetDefaultReindexExecutionMode`

**Comment by xbrianlee** [comment]
@CaiocSiqueira Can you please make the change above and then forward your changes? Thanks!

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:forward

**Pull request closed without merging**